### PR TITLE
First working implementation

### PR DESCRIPTION
**Pull request by rossmeier**
TODO:
 * [x] Testing
 * [x] Unit tests

### CODE REVIEW

File: db/schema.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	gametype tiny,
+	points tiny,
+	schwarz bit,
+
+	time datetime,
+	reporter int
+);`
+
+type dbUser struct {
+	ID       int
```
Comment (thielpa) [inline db/schema.go:35]: Why ID in complete capital?

File: db/statements.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+func initStatements() error {
+	var err error
+
+	getExactUserStatement, err = db.Preparex("SELECT * FROM users WHERE LOWER(username)=LOWER(?)")
+	if err != nil {
+		return errors.Wrap(err, "Error preparing getExactUserStatement")
+	}
+
+	insertGameStatement, err = db.PrepareNamed(`INSERT INTO games (part0, part1, part2, part3, player, playmate, gametype, points,	schwarz, time, reporter)
```
Comment (thielpa) [inline db/statements.go]: Strange tab

File: db/static.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	"github.com/jmoiron/sqlx"
+	"github.com/pkg/errors"
+)
+
+var db *sqlx.DB
+
+// InitDB initializes the database
+func InitDB() error {
+	var err error
+	//db = sqlx.Connect("mysql", "root:root@localhost/db")
```
Comment (thielpa) [inline db/static.go:16]: remove before merging

File: main.go [hunk 4]
```diff
@@ -1,12 +1,42 @@
...
+
+	// HTTP root serves html page
 	r.GET("/", routes.Root)
+
+	// Login route
+	r.POST("/login", routes.Login)
+
+	// Routes requiring login
+	g := r.Group("/", middleware.GetUser)
+	/**/ g.POST("/newGame", routes.NewGame)
```
Comment (thielpa) [inline main.go:28]: What is the comment for?

File: db/schema.go (hunk 1 above)
Comment (rossmeier) [reply db/schema.go:35]: go convention

File: db/statements.go (hunk 2 above)
Comment (rossmeier) [reply db/statements.go]: done

File: db/static.go (hunk 3 above)
Comment (rossmeier) [reply db/static.go:16]: I would keep it in since it is temp anyways

File: main.go (hunk 4 above)
Comment (rossmeier) [reply main.go:28]: Force indentation after auto-format

**Review by thielpa — APPROVED** [review]

**Pull request merged**